### PR TITLE
[burn-staging] The download full schedule box animates at a different time than the rest of the sidebar

### PR DESCRIPTION
**Pull request by dmitrii-zolotukhin**
- [x] Fixed bug: the download full schedule box animates at a different time than the rest of the sidebar

fixed sparkletown/internal-sparkle-issues#1166

### CODE REVIEW

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 1]
```diff
@@ -305,25 +305,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
-          {isShowPersonalDownloadBtn && (
+        {!isEventsLoading && (
+          <div className="NavBarSchedule__downloadButtons">
+            {isShowPersonalDownloadBtn && (
+              <Button
+                onClick={downloadPersonalEventsCalendar}
+                customClass="NavBarSchedule__downloadScheduleBtn"
+              >
+                Download your schedule
+              </Button>
```
Comment (yurii-lubynets) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:316]: Could we use `<ButtonNG/>` component here?

Comment (goran-peoski-work) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:316]: Any instance where `Button` can be replaced with `ButtonNG` that doesn't require significant effort for adjustment is a plus. The sooner we eliminate the need of  `Button`, the sooner we can get rid of some `global.scss` styles that are hard to override.

File: src/components/organisms/NavBarSchedule/NavBarSchedule.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import "scss/constants.scss";
 
-$NavBarSchedule--padding: 20px;
+$navBarSchedule--padding: 20px;
```
Comment (yurii-lubynets) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.scss:3]:
```suggestion
$navbar-schedule-padding: 20px;
```

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx (hunk 1 above)
Comment (dmitrii-zolotukhin) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fixed in 455ec1701e65eecc756037aea9c0f4502148a8e7

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 3]
```diff
@@ -305,25 +305,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
...
+            )}
             <Button
-              onClick={downloadPersonalEventsCalendar}
-              customClass="NavBarWrapper__download-schedule-btn"
+              onClick={downloadAllEventsCalendar}
+              customClass="NavBarSchedule__downloadScheduleBtn"
             >
-              Download your schedule
+              Download full schedule
             </Button>
```
[the remark this answers is not in the document]
Comment (dmitrii-zolotukhin) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx]: fixed in 455ec17

File: src/components/organisms/NavBarSchedule/NavBarSchedule.scss (hunk 2 above)
Comment (dmitrii-zolotukhin) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.scss]: fixed in 455ec17

**Comment by ermiaSparkle** [comment]
We dont want this in burn staging anymore

**Pull request closed without merging**